uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/manager-leave/components/manager-leave-container.js

```diff
@@ -9,12 +9,14 @@ define([
       return settings.pathTpl + 'components/manager-leave-container.html';
     }],
     controllerAs: 'managerLeave',
-    controller: ['$log', function ($log) {
+    controller: ['$log', '$rootScope', function ($log, $rootScope) {
       console.log('CRM id of the currently logged in user: ', this.contactId);
       $log.debug('Component: manager-leave-container');
 
       var vm = Object.create(this);
 
+      $rootScope.isManager = true;
```

Comment:
When we have a way to know how admin can be recognised, we can set another variable like this, and using that same variable request-ctrl can decide, which role it is.